docs/themes/theme_settings.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Writing Theme Settings
+---
+
+You can provide users with theme settings, configurable parts of your theme that help users customize the theme to their projects. Examples of theme settings include configurable element colors, image asset uploads, footer content.
+
+You create the form fields for your theme settings in `config/theme.ini`. Start with `[config]` as the heading under the `[info]` metadata. The `[config]` heading begins a list of theme settings; you do not need to include it over each individual setting.
+
+```
+[info]
+name = "Theme Name"
+,,,
```

Comment:
Supposed to be an ellipsis?

also, even in an example, we generally want to put that kind of thing that's not "really" part of the syntax as a comment, so line-commented with `#` in this case